Makefile

```diff
@@ -112,7 +115,7 @@ dockerenvbuild: writeversion
 dockerenvdist: writeversion
 	docker run -v $$PWD/dist:/kolibridist learningequality/kolibri:$$(cat kolibri/VERSION)
 
-BUMPVERSION_CMD = bumpversion --current-version `python -m kolibri --version` $(PART_INCREMENT) --allow-dirty -m "new version" --no-commit --list
+BUMPVERSION_CMD = bumpversion --current-version `python -c "import kolibri; print(kolibri.__version__)"` $(PART_INCREMENT) --allow-dirty -m "new version" --no-commit --list
```

Comment:
There's no real difference here. I don't remember why I did it. But it also works :)